components/wso2is.key.manager/src/main/java/org/wso2/is/client/WSO2ISConstants.java

```diff
@@ -24,6 +24,10 @@ public class WSO2ISConstants {
 
     public static final String WSO2IS_TYPE = "WSO2-IS";
     public static final String DISPLAY_NAME = "WSO2 Identity Server";
+    public static final String KM_ADMIN_AS_APP_OWNER_NAME = "km_admin_as_app_owner";
+    public static final String KM_ADMIN_AS_APP_OWNER_LABEL =
+            "Enable admin user as the owner of created OAuth applications";
```

Comment:
Where is this displayed? Can u attach a screenshot of the admin keymanager ui to the PR plz

Reply:
Updated the PR description with a screenshot